students/drovdahl/lesson07/html_render.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+Ryan Drovdahl's version
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+
+    # 'html' is the default tag that is only used for testing
+    tag = 'html'
+    close_tag_line = True
+    link_tag = False
+    meta_tag = False
+
+    def __init__(self, content=None, style=None, id=None, indent=4):
```

Comment:
`style` and `id` are not the only attributes that may be specified in an HTML element. Therefore, you should have received `**kwargs` to accommodate arbitrary attributes.